### ISSUE TITLE
Follow authors or categories?

### ISSUE THREAD

**Issue by poVoq** [jdormit/pterotype#20]
Is there a plan to allow other contacts then blog@?

Would be cool if individual authors or categories could get their own.

Thanks!

**Comment by jdormit** [comment]
Yup, that is in the roadmap. In Pterotype V2, every WordPress user will
have their own feed, and users with the right permissions will have their
posts published to the blog feed as well.
